.github/workflows/release_deploy_windows.yml

```diff
@@ -2,6 +2,9 @@ name: Publish MSIX to Microsoft Store
 
 on: workflow_dispatch
 
+# Declare default permissions as read only.

```

Comment:
This comment seems a bit redundant, feel free to remove it if you'd like. I don't feel strongly :)